tests/core/data/layer/v1/sqlalchemy/__init__.py

```diff
@@ -0,0 +1,13 @@
+from mongomock_motor import AsyncMongoMockClient, AsyncMongoMockCollection
+
+
+async def init_mongo(
+    db_name: str, collection: str, cleanup: bool = True
+) -> AsyncMongoMockCollection:
+    client = AsyncMongoMockClient(
+        "mongodb://user:pass@host:27017", connectTimeoutMS=250
+    )
+    mongo_collection: AsyncMongoMockCollection = client[db_name][collection]
+    if cleanup is True:
+        await mongo_collection.delete_many({})
+    return mongo_collection
```

Comment:
Why is this snippet exactly the same as the one in `tests.core.data.layer.v1.mongo`?  Couldn't we simply import it from there?

Reply:
Removed